src/components/Toolbar.js

```diff
@@ -10,13 +54,62 @@ const shortcutIds = [
   'search',
 ]
 
-export const Toolbar = connect(({ settings: { dark } }) => ({ dark }))(({ dark }) =>
-  <div className="toolbar">
-    {shortcutIds.map(id => {
-      const { name, svg: Icon, exec } = shortcutById(id)
-      return <div key={id} className="toolbar-icon" title={name} onClick={exec}>
-        <Icon fill={dark ? 'white' : 'black'} />
-      </div>
-    })}
-  </div>
-)
+const ToolbarMessageOverlay = ({ classname = 'toolbar-overlay', name, description }) => {
+    return <TransitionGroup>
+            <CSSTransition key={0} timeout={200} classNames='fade'>
```

Comment:
The `TransitionGroup` and `CSSTransition` need to go outside the show/hide logic to work. i.e. outside the `null` render condition ([line 108](https://github.com/cybersemics/em/pull/265/files#diff-e007c62f341ba9b8108617f3d82d21c8R108)).

Reply:
Got it.